src/sip/request.c

```diff
@@ -207,6 +211,30 @@ static int request(struct sip_request *req, enum sip_transp tp,
 }
 
 
+static int dst_set_scopeid(struct sa *dst, struct sip_request *req)
```

Comment:
I am not sure if the application should have knowledge about scope id.

this should be done on a low-level like in sa or net

Reply:
The scope ID has to be copied from chosen `laddr` to the destination address. @juha-h and me have already thought about choosing the `laddr` in a very low level. But it has to be put into the SDP.  So in order to make IPv6ll work:
- We have a destination IPv6ll address.
- Loop over all local IPv6ll addresses.
- Copy scope ID from local address to destination address.
- Test if there is a route from local to destination address.
- The found `laddr` has to be put into the SDP.
- Then the destination address has the correct scope ID.